pkg/ansible/runner.go

```diff
@@ -129,10 +203,8 @@ func (r *ansibleRunner) RunPlaybook(clusterName, jobPrefix, playbook, inventory,
 					},
 				},
 
-				// TODO: possibly drop this once https://github.com/openshift/openshift-ansible/pull/6320 merges, the default run script should then work, minus the vars file :(
-				//Command: []string{"ansible-playbook", "-i", "/ansible/inventory/hosts", "--private-key", "/ansible/ssh/privatekey.pem", playbookPath, "-e", "@/ansible/inventory/vars"},
-
-				// TODO: committing as no-op for now, uncomment command above to start testing provisioning
+				// TODO: committing as no-op for now, uncomment to use the default
+				// origin-ansible container entrypoint and actually run playbooks.
```

Comment:
I am confused about what is supposed to be uncommented here? I thought it was the previous line that has been removed.
```
Command: []string{"ansible-playbook", "-i", "/ansible/inventory/hosts", "--private-key", "/ansible/ssh/privatekey.pem", playbookPath, "-e", "@/ansible/inventory/vars"},
```

Reply:
Should this be that the next line should be **commented out**?

Reply:
You just uncomment the explicit Command and let it fall back to the default container entrypoint. One of our run as root problems was already solved in their entry scripts, and originally we found and fixed a bug but they've got that merged and in the container now. The env vars above control all the options we were doing in previous iteration of the Command. 

Wording isn't good here, should be "comment out" not uncomment. Fixing now.